rewrite-java/src/main/resources/META-INF/rewrite/static-analysis-cleanup.yml

```diff
@@ -20,53 +20,64 @@ displayName: Common static analysis issues
 description: Resolve common static analysis issues discovered through 3rd party tools
```

Comment:
can you go through each of the commented-out recipes and make a determination if these should just always be part of the auto-remediation of static analysis issues?  

The only one that I know that should NOT be on this list is the `FixSerializableFields`, this really needs some human intervention. 

If you have questions on the others, just do a git blame and ask the original author if it was just an oversight for it not being included.

Reply:
sure.  thanks @tkvangorder